README.md

```diff
@@ -134,9 +134,9 @@ controller_server:
   ros__parameters:
     FollowPath:
       plugin: "mppi::Controller"
-      time_steps: 15
-      model_dt: 0.1
-      batch_size: 300
+      time_steps: 40
+      model_dt: 0.075
+      batch_size: 1000
       vx_std: 0.2
       vy_std: 0.2
       wz_std: 1.0
```

Comment:
0.3 - 0.5

Reply:
Covered by @padhupradheep comment about 0.4 to use